csrankings-m.csv

```diff
@@ -420,6 +423,7 @@ Marcelo R. Pias,Federal University of Rio Grande,https://sites.google.com/site/p
 Marcelo Risk,University of Buenos Aires,https://www.dc.uba.ar/rrhh/profesores/risk,NOSCHOLARPAGE
 Marcelo Soares Pimenta,UFRGS,http://inf.ufrgs.br/~mpimenta,y7D-szYAAAAJ
 Marcelo Walter,UFRGS,http://www.inf.ufrgs.br/~mwalter,7VfUEj0AAAAJ
+Marcelo	Worsley,Northwestern University,https://sites.northwestern.edu/marceloworsley/,LXV6vHgAAAAJ

```

Comment:
Not in CS.

Reply:
He has a joint appointment in CS (named chair) and Learning Sciences. His website and CV that you linked say this. He is also in our TT faculty list (see below). Why is he "not in CS"?

 https://www.mccormick.northwestern.edu/computer-science/people/faculty/tenure-track.html

Reply:
That wasn't clear (homepage says "Associate Professor, Learning Sciences"). Thanks for the clarification.